datalad/interface/run.py

```diff
@@ -626,7 +636,7 @@ def run_command(cmd, dataset=None, inputs=None, outputs=None, expand=None,
                          'cannot detect changes by command'))
             return
 
-    cmd = normalize_command(cmd)
+    cmd = assure_unicode(normalize_command(cmd))
```

Comment:
why this `assure_unicode` is needed if `normalize_command` already does assure_unicode in both list and non-list cases?

Reply:
guh, it's not.  I forgot to remove it.  Thanks.